src/main/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -319,6 +319,32 @@ public void datasetsInformation() {
     if (dmp.getTargetAudience() != null)
       addReplacement(replacements, "[targetaudience]", dmp.getTargetAudience());
     else addReplacement(replacements, "[targetaudience]", "");
+
+    StringBuilder reusedDescription = new StringBuilder();
+    List<Dataset> reusedDatasets = getReusedDatasets();
+    for (int i = 0; i < reusedDatasets.size(); i++) {
+      Dataset dataset = reusedDatasets.get(i);
+      if (dataset.getDescription() != null && !dataset.getDescription().isEmpty()) {
+        reusedDescription.append("Description for \"");
+        reusedDescription.append(dataset.getTitle()).append("\": ;");
+        reusedDescription.append(dataset.getDescription()).append(";");
+      }
+    }
+
+    addReplacement(replacements, "[reuseddatadescription]", reusedDescription.toString());
+
+    StringBuilder newDescription = new StringBuilder();
+    List<Dataset> newDatasets = getNewDatasets();
+    for (int i = 0; i < newDatasets.size(); i++) {
+      Dataset dataset = newDatasets.get(i);
+      if (dataset.getDescription() != null && !dataset.getDescription().isEmpty()) {
+        newDescription.append("Description for \"");
+        newDescription.append(dataset.getTitle()).append("\": ;");
+        newDescription.append(dataset.getDescription()).append(";");
+      }
```

Comment:
Sooner or later, someone will probably be like: `Why is there no description exported for dataset x/y` :smile: 

Might make sense to add an empty string, if the description is null or empty.